### DISCUSSION TITLE
Renaming master to main

### DISCUSSION THREAD

**Discussion by j-bennet** [dbcli/pgcli#1316] [Announcements]
We renamed `master` branch of `pgcli` to `main`.

If you have a local clone of `pgcli` where main branch is still `master`, you'll have to do the following:

```
git branch -m master main
git fetch origin
git branch -u origin/main main
git remote set-head origin -a
```

cc @dbcli/pgcli-core.

**Comment by laixintao** [comment]
LGTM

**Comment by dbaty** [comment]
Renaming happened 2 years ago, I am closing this (I guess it was an announcement more than a discussion).

**Reply by laixintao** [reply]
Yes, we are on `main` now.
